lua/dylib.lua

```diff
@@ -30,6 +50,48 @@ int add_fib_entry(const char *prefix, const char *gt_ip, const char *gw_ip,
 	enum gk_fib_action action, struct gk_config *gk_conf);
 int del_fib_entry(const char *ip_prefix, struct gk_config *gk_conf);
 
+struct gk_fib_dump_entry *list_fib_entries(struct gk_config *gk_conf,
+	uint32_t *num_entries);
+
 ]]
 
 c = ffi.C
+
+--
+-- Listing GK FIB entries functions
+--
+
+function lua_list_fib_entries(entry_tbl, num_entries)
+	local tbl = ffi.cast("struct gk_fib_dump_entry *", entry_tbl)
```

Comment:
If `entry_tbl` is a lot of memory, this function will allocate even more memory.